WORKSPACE

```diff
@@ -1,18 +1,13 @@
-workspace(name = "root")
+workspace(name = "libsbp")
 
-new_local_repository(
-    name = "my-check",
-    build_file = "bazel/check.BUILD",
-    path = "c/third_party/check",
-)
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-local_repository(
-    name = "my-googletest",
-    path = "c/third_party/googletest",
+http_archive(
+    name = "rules_swiftnav",
+    strip_prefix = "rules_swiftnav-b80ca392ea24a14f8d810380b14495680255903b",
```

Comment:
I guess the downside is this doesn't (yet) hook in to our submodule update machinery?

Reply:
Theres this project called [renovate](https://github.com/swift-nav/gnss-converters-private/pull/1368) that's been [recommended](https://github.com/hedronvision/bazel-compile-commands-extractor#suggestion-updates) for this purpose.

I know we use dependabot, but supposedly it has really good bazel support

Reply:
https://github.com/renovatebot/renovate/blob/main/docs/usage/bazel.md

Reply:
https://github.com/dependabot/dependabot-core/issues/2196